setup.py

```diff
@@ -92,7 +92,11 @@
     packages=find_packages(where='src'),
     package_dir={'': 'src'},
     scripts=['scripts/garage'],
-    python_requires='>=3.5',
+    # DeepMind has not yet uploaded a pre-compiled version of dm-tree to
+    # PyPI for Python 3.8, causing PIP to compile it from source. This fails
+    # without bazel, so for now, Python 3.8 support is disabled by default.
+    # See: https://github.com/deepmind/dm_control/issues/135
```

Comment:
I think you should link to https://github.com/deepmind/tree/issues/1 which will be closed once they update the wheels. I have reason to believe the Python 3.8 wheels will appear soon.

Reply:
Even better, just link the to garage issue for this and keep it open. That way we will notice that our own is open and can know to check whether to close it.